apisix/core/config_etcd.lua

```diff
@@ -186,7 +186,7 @@ local function load_full_data(self, dir_res, headers)
         end
 
         if data_valid and self.checker then
-            data_valid, err = self.checker(item.value)
+            data_valid, err = self.checker(item.value, item.key)
```

Comment:
We should also update `config_yaml.lua` & add a test in https://github.com/apache/apisix/blob/master/t/config-center-yaml/plugin-metadata.t